ddtrace/contrib/tornado/handlers.py

```diff
@@ -104,7 +104,7 @@ def log_exception(func, handler, args, kwargs):
         # is not a 2xx. In this case we want to check the status code to be sure that
         # only 5xx are traced as errors, while any other HTTPError exception is handled as
         # usual.
-        if 500 <= value.status_code <= 599:
+        if config.http_server.is_error_code(value.status_code):
```

Comment:
By default `http_server.is_error_code()` matches integers from 500 to 599. This change should not change the default behavior of tornado error handlers.